ideascube/tests/helpers.py

```diff
@@ -0,0 +1,37 @@
+from django.apps import apps
+from django.test import TransactionTestCase
+from django.db.migrations.executor import MigrationExecutor
+from django.db import connection
+import pytest
+
+
+@pytest.mark.skipif(
+    pytest.config.getoption("--nomigrations"),
+    reason="skipping due to --nomigrations"
+)
+class MigrationTestCase(TransactionTestCase):
```

Comment:
I think that's something you can do with request fixture and indirect params: http://docs.pytest.org/en/latest/example/parametrize.html#apply-indirect-on-particular-arguments